### DISCUSSION TITLE
How to apply DesktopAcrylicBackdrop to a uno built window

### DISCUSSION THREAD

**Discussion by cconner100** [unoplatform/uno#19123] [Q&A] [answered]
Using the window created by Builder.Window i am trying in Windows only to apply a DesktopAcrylicBackdrop;

I am using the Window returned by Builder.Window then settings the window.SystemBackdrop = new DesktopAcrylicBackdrop(); which seems to have no effect at all.

In addition when i check to see what type the window is, it is not a WindowEx which is needed for the SystemBackdrop to work?

Any ideas

**Comment by morning4coffe-dev** [comment, marked as answer]
Hey @cconner100,

When you say "Uno built window," are you referring to specific platforms? If you mean just WinUI (Windows), you can use `MainWindow.SystemBackdrop`. You can check out my [project](https://github.com/morning4coffe-dev/recurrents/blob/05d99d3a13169e3afe2d94e09b4b0063e70878e4/src/Recurrents/App.xaml.cs#L100C1-L103C10) for more details. Make sure to set all the backgrounds to `Transparent`. 

I recommend creating a platform-specific `Brush` and then applying it. You can see an [example here](https://github.com/morning4coffe-dev/recurrents/blob/05d99d3a13169e3afe2d94e09b4b0063e70878e4/src/Recurrents/App.xaml#L36C1-L39C54).

For Skia (desktop), it’s more complicated. You would either need to implement it on your side or use specific APIs for the window. However, I’m not sure if this is possible with the WPF window.

Let me know if this helps!
